corner 2.2.2 compatibility

**Describe the bug**
With [corner 2.2.2](https://pypi.org/project/corner/2.2.2/), we get test failures like [here](https://github.com/PyFstat/PyFstat/actions/runs/4643674598/jobs/8218439000#step:8:3240):
```
FAILED PyFstat/tests/test_mcmc_based_searches.py::TestMCMCTransientSearch::test_transient_MCMC_t0_tau_BtSG - ValueError: 'title_quantiles' must contain exactly three values; pass a length-3 list or array using the 'title_quantiles' argument
```

--- a/examples/glitch_examples/PyFstat_example_glitch_robust_directed_MCMC_search_on_1_glitch.py
+++ b/examples/glitch_examples/PyFstat_example_glitch_robust_directed_MCMC_search_on_1_glitch.py
@@ -94,7 +94,7 @@ logger.info("Making corner plot...")
 mcmc.plot_corner(
     label_offset=0.25,
     truths={"F0": F0, "F1": F1, "delta_F0": delta_F0, "tglitch": tstart + dtglitch},
-    quantiles=(0.16, 0.84),
+    quantiles=(0.16, 0.5, 0.84),
     hist_kwargs=dict(lw=1.5, zorder=-1),
     truth_color="C3",
 )

--- a/pyfstat/mcmc_based_searches.py
+++ b/pyfstat/mcmc_based_searches.py
@@ -1065,7 +1065,7 @@ class MCMCSearch(BaseSearchClass):
                 hist_kwargs=hist_kwargs,
                 show_titles=True,
                 fill_contours=True,
-                quantiles=[0.05, 0.95]
+                quantiles=[0.05, 0.5, 0.95]  # [lower, central, upper]
                 if "quantiles" not in kwargs
                 else kwargs.pop("quantiles"),
                 verbose=True if "verbose" not in kwargs else kwargs.pop("verbose"),

